### ISSUE TITLE
Wrong state with Exoplayer

### ISSUE THREAD

**Issue by dbof10** [android/uamp#358]
I try to update with Exoplayer 2.11.4 current version 2.10.0. When I click play a music, it's paused  after buffering. I put a log here

```
  override fun onPlaybackStateChanged(state: PlaybackStateCompat?) {
            Log.e("ABCD", "state " + state?.stateName)

            playbackState.postValue(state ?: EMPTY_PLAYBACK_STATE)
       }
```
and receive some following logs

```
2020-05-23 11:51:54.731 2719-2719/com.example.android.uamp.next E/ABCD: state STATE_BUFFERING
2020-05-23 11:51:54.752 2719-2719/com.example.android.uamp.next E/ABCD: state STATE_BUFFERING
2020-05-23 11:51:54.754 2719-2719/com.example.android.uamp.next E/ABCD: state wake_up_01
2020-05-23 11:51:55.396 2719-2719/com.example.android.uamp.next E/ABCD: state STATE_BUFFERING
2020-05-23 11:51:55.405 2719-2719/com.example.android.uamp.next E/ABCD: state wake_up_01
2020-05-23 11:51:55.456 2719-2719/com.example.android.uamp.next E/ABCD: state STATE_PAUSED
```

**Comment by dturner** [comment]
Just merged the update to ExoPlayer 2.11.5. Please could you retest and let me know if you're still experiencing this issues.

**Comment by dbof10** [comment]
Seems to work for me
